rules-reviewed/openshift/java-rpc.windup.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<ruleset id="java-rpc" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This is a ruleset for Java Remote Procedure Call specific rules for migrating to OpenShift.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
+        </dependencies>
+        <sourceTechnology id="java" />
+        <sourceTechnology id="rpc" />
+        <targetTechnology id="cloud-readiness" />
+        <tag>rpc</tag>
+    </metadata>
+    <rules>
+        <rule id="java-rpc-00000">
+            <when>
+                <or>
+                    <javaclass references="javax.xml.rpc{*}" as="default">
+                        <location>CONSTRUCTOR_CALL</location>
+                        <location>IMPORT</location>
+                        <location>INHERITANCE</location>
+                        <location>TYPE</location>
+                        <location>VARIABLE_DECLARATION</location>
+                    </javaclass>
+                    <javaclass references="org.apache.xmlrpc{*}" as="default">
+                        <location>CONSTRUCTOR_CALL</location>
+                        <location>IMPORT</location>
+                        <location>INHERITANCE</location>
+                        <location>TYPE</location>
+                        <location>VARIABLE_DECLARATION</location>
+                    </javaclass>
+                    <javaclass references="redstone.xmlrpc{*}" as="default">
+                        <location>CONSTRUCTOR_CALL</location>
+                        <location>IMPORT</location>
+                        <location>INHERITANCE</location>
+                        <location>TYPE</location>
+                        <location>VARIABLE_DECLARATION</location>
+                    </javaclass>
+                </or>
+            </when>
+            <perform>
+                <iteration>
+                    <hint title="Java Remote Procedure Call (RPC)" effort="5" category-id="cloud-mandatory">
```

Comment:
I think the effort level assignment here is too low. I think 7 is closer to reality, as changing out an XMLRPC system for something else requires a major architectural decision.